src/constants.rs

```diff
@@ -56,7 +57,7 @@ pub const KRILL_CLI_SERVER_ARG: &str = "server";
 pub const KRILL_CLI_SERVER_ENV: &str = "KRILL_CLI_SERVER";
 pub const KRILL_CLI_SERVER_DFLT: &str = "https://localhost:3000/";
 
-pub const KRILL_CLI_TOKEN_ARG: &str = "token";
+pub const KRILL_CLI_ADMIN_TOKEN_ARG: &str = "token";
```

Comment:
Should this contain _ADMIN? Doesn't it's naming follow the same rationale as that of `KRILL_CLI_TOKEN_ENV`?